client/app/widgets/Resources/index.jsx

```diff
@@ -110,28 +118,53 @@ export class Resources extends React.Component<Props, State> {
     });
   };
 
+  onClick = () => {
+    const { resources } = this.props;
+    this.setState((prevState: State) => ({
+      resourcesDisplayed: prevState.resourcesDisplayed + RESOURCES_PER_PAGE,
+      lastPage:
+        prevState.resourcesDisplayed + RESOURCES_PER_PAGE >= resources.length,
+    }));
+  };
+
+  displayLoadMoreButton = () => (
+    <center>
+      <button
+        type="button"
+        className={`loadMore ${css.buttonDarkM}`}
+        onClick={this.onClick}
+      >
+        {I18n.t('load_more')}
+      </button>
+    </center>
+  );
+
   displayTags = () => {
-    const { checkboxes } = this.state;
+    const { checkboxes, resourcesDisplayed, lastPage } = this.state;
+    const { resources } = this.props;
     const filteredResources = this.filterList(checkboxes);
     return (
       <>
         <center className={css.marginTop}>
-          {`${filteredResources.length} ${I18n.t(
-            'navigation.resources',
-          ).toLowerCase()}`}
+          {`${Math.min(resourcesDisplayed, resources.length)} of ${
```

Comment:
We'll want to translate "of". I recommend creating a key in our translation files like so:
```
of: "%{first} of %{second}"
```

You can use Google translate for the other languages.